### PR TITLE
Added Geolite ASN database parsing.

### PR DESCRIPTION
**Pull request by lavalamp-**
Hi there!

Thank you all so much for your work - I've been a long-time fan.

I added some functionality to parse / process the Geolite ASN database from MaxMind. Looks pretty straightforward and it works locally, but if you'd like me to add anything else just lmk!

### CODE REVIEW

**Comment by zakird** [comment]
Do you have an example output? It's difficult to tell what this looks like offhand because of the `interface{}` return just handing back the library object.

File: geoipasn.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+	return nil
+}
+
+func (anno *GeoIPASNAnnotator) GetFieldName() string {
+	return "geoasn"
+}
+
+func (anno *GeoIPASNAnnotator) Annotate(ip net.IP) interface{} {
+	record, err := anno.Reader.ASN(ip)
+	if err != nil {
```
Comment (zakird) [inline geoipasn.go:86]: What happens when reader just can't look up an IP. It seems nonoptimal to panic on a single lookup that fails.

Comment (lavalamp-) [reply geoipasn.go:86]: Yeah - good call out. Will fix.

**Comment by lavalamp-** [comment]
> Do you have an example output? It's difficult to tell what this looks like offhand because of the `interface{}` return just handing back the library object.

I'll update to have an explicitly populated struct, similar to how `geoip2.go` does.

**Comment by lavalamp-** [comment]
Output of new changes:

```
chris@ubuntu:~/git/zannotate$ cat test.txt  | ./main --geoasn --geoasn-database=geolite_asn.mmdb
{"ip":"8.8.8.8","geoasn":{"asn":15169,"asn_org":"GOOGLE"}}
{"geoasn":{"asn":13335,"asn_org":"CLOUDFLARENET"},"ip":"1.1.1.1"}
```

**Comment by lavalamp-** [comment]
Done!

```
chris@ubuntu:~/git/zannotate$ cat test.txt | ./main --geoasn --geoasn-database=geolite_asn.mmdb
{"ip":"8.8.8.8","geoasn":{"asn":15169,"org":"GOOGLE"}}
{"ip":"1.1.1.1","geoasn":{"asn":13335,"org":"CLOUDFLARENET"}}
```

**Pull request merged**